src/app/_components/FilterContainer.tsx

```diff
@@ -6,18 +6,18 @@ type LayoutProps = {
   children: ReactNode
 }
 
-type DesktopFilterProps = {
+type FilterComponents = [ReactNode, ReactNode?]
+
+type DesktopFiltersProps = {
   search: ReactNode
-  location?: ReactNode
+  filters?: FilterComponents
```

Comment:
@CharlyMartin - I remember we discussed wanting to have clearer naming for `filters` prop since in its current state it could be ambiguous - it might refer to filter values/data rather than the UI components.

Would this be the right place to change it? To something like `filterComponents`?

Reply:
Yes we can have `filterComponents`, `searchComponent` and `sortComponent`